src/components/StudyInfo/CavaticaCounts.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {List, Icon, Popup, Label, Button} from 'semantic-ui-react';
+import {projectOptions} from '../../common/enums';
+
+const CavaticaCounts = ({projects, title}) => {
+  if (projects && projects.length > 0) {
+    const types = projects.map(({node: {projectType}}) => projectType);
+    const typeCounts = types.reduce((count, type) => {
+      count[type] = (count[type] || 0) + 1;
+      return count;
+    }, {});
+    return (
+      <List horizontal>
```

Comment:
minor thing but you could prob move this up and then just conditionally render just the list items. Also curious if we need to use a list here

Reply:
Using the list would have the dot, icon, and text in a consistent style, which we can reuse a similar style somewhere else too.